build_tools/ci/run_matmul_test.sh

```diff
@@ -782,6 +782,21 @@ if [ -d "$VITIS" ]; then
       --num_repeat_runs "10" \
       --use_ukernel "1"
 
+# Strix ukernel chess test
+  run_matmul_test \
+      --name_prefix "chess_bf16_f32_npu4_ukernel" \
+     --target_device "npu4" \
```

Comment:
This is not going to work, because the machine in CI is not strix. I can try adding it today, but it might be a bit cumbersome to support both strix and phoenix within the same bash script as some tests will only work on phoenix and others only on strix. The python tests should be easier for that.